test/format-test.cc

```diff
@@ -1788,7 +1788,8 @@ FMT_BEGIN_NAMESPACE
 template <> struct formatter<point> : nested_formatter<double> {
   auto format(point p, format_context& ctx) const -> decltype(ctx.out()) {
     return write_padded(ctx, [this, p](auto out) -> decltype(out) {
-      return format_to(out, "({}, {})", nested(p.x), nested(p.y));
+      // Namespace-qualify to avoid ambiguity with std::format_to
```

Comment:
I don't think this comment is needed because we have a bunch of such calls and it doesn't make sense to explain conflict with std (with is a known issue in general) on one random call. Please remove.

Reply:
Ok, I have removed the comment. Thanks for reviewing.